prototype/mobile_endpoint/dao.py

```diff
@@ -132,6 +174,34 @@ def get_reverse_indexed_cases(self, domain, case_ids):
             .options(
                 contains_eager('indices'),
                 defer(CaseData.case_json)
+        ).all()
+
+    def get_open_case_ids(self, domain, owner_id):
+        return [row[0] for row in CaseData.query.with_entities(CaseData.id).filter(
```

Comment:
I'm not familiar enough with sqlalchemy to know whether this syntax is smart enough just just pull IDs from the DB. do you know / is it?